js/app.js

```diff
@@ -1,9 +1,26 @@
 // wait for DOM to load before running JS
 $(document).ready( function() {
-
```

Comment:
Hey Ellis,

This code is perfect. Great job as always. 

I would encourage you to try the bonuses next time or try adding some other features that aren't in the assignment if you get done with the homework quickly. 

Matt